blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,35 @@
+![Knative Booth](images/virtual-office-hour-12-2022-with-text.jpg)
+# First 🌱 Knative Virtual Office Hour
```

Comment:
```suggestion
# First 🌱 Knative Virtual Office Hour

![Knative Booth](images/virtual-office-hour-12-2022-with-text.jpg)
```

I think due to how mkdocs renders the page, the first line needs to be the title, otherwise it uses the filename on the left-side menu.

Reply:
Moved to safe location - hopefully mkdocs will be happy